TombEngine/Scripting/Internal/TEN/Objects/ObjectsHandler.h

```diff
@@ -64,6 +64,45 @@ class ObjectsHandler : public ScriptInterfaceObjectsHandler
 			return std::make_unique<R>(std::get<R::IdentifierType>(m_nameMap.at(name)));
 	}
 
+	template <typename R, char const* S>
+	std::vector <std::unique_ptr<R>> GetMoveablesBySlot(GAME_OBJECT_ID objID)
+	{
+		std::vector<std::unique_ptr<R>> items = {};
+		for (auto& [key, val] : m_nameMap)
+		{
+			if (!std::holds_alternative<std::reference_wrapper<MESH_INFO>>(val))
+			{
+				auto* item = &g_Level.Items[GetIndexByName(key)];
+				GAME_OBJECT_ID objectNumber = item->ObjectNumber;
+
+				if (objID == objectNumber)
```

Comment:
If objectNumber is only used once then it will be cleaner to test `objID == item->ObjectNumber` directly

Reply:
done